### DISCUSSION TITLE
Provide screen name for user who edited the entry last

### DISCUSSION THREAD

**Discussion by matthewcarleton** [ExpressionEngine/ExpressionEngine#2877] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When showing entry information you can show the author, publish date, and recent edit date but you can not say who made the most recent edit.

**Describe the solution you'd like**
I'd like to be able to list who is responsible for the most recent edit.

**Describe alternatives you've considered**
Any workarounds for this would be a lot of effort on the publisher/editor and wouldn't be worth it.

**Teachability, Documentation, Adoption, Migration Strategy**
{last-editor}
The screen name of the last user to edit the entry.

**Comment by matthewcarleton** [comment]
here's the query to show the last editor and their avatar?
`{exp:query sql="SELECT exp_members.screen_name AS revision_author, exp_entry_versioning.version_date, exp_members.avatar_filename as avatar FROM exp_entry_versioning JOIN exp_members ON exp_entry_versioning.author_id=exp_members.member_id WHERE entry_id='{entry_id}' ORDER BY version_date DESC LIMIT 1"}`

**Comment by shbchk** [comment]
Here is my query for just a screen name:

```
{exp:query sql="
  SELECT em.screen_name AS last_edited_by_screen_name
  FROM   exp_entry_versioning AS ev
  JOIN   exp_members            AS em ON em.member_id = ev.author_id
  WHERE  ev.entry_id = {entry_id}
  ORDER  BY ev.version_date DESC
  LIMIT  1
"}{last_edited_by_screen_name}{/exp:query}
```
